onnx/defs/nn/defs.cc

```diff
@@ -102,8 +102,8 @@ namespace onnx {
                         pads_doc.c_str(),
                         AttrType::INTS);
             schema.Attr("p",
-                        "p value of the Lp norm used to pool over the input data, default is 2.0.",
-                        AttrType::FLOAT);
+                        "p value of the Lp norm used to pool over the input data, default is 2.",
+                        AttrType::INT);
```

Comment:
I think this is a also a breaking change.

Reply:
Sure, will bump the version.